### DISCUSSION TITLE
What happened to Realm Doc?

### DISCUSSION THREAD

**Discussion by xoex** [realm/realm-java#7555] [General]
I have to admit, one of many things I loved about Realm was its documentary.
It was very straightforward, compact, readable and well designed.

In opposition, MongoDB's documentaries (like MongoDB doc about their own database) is just pain in ***. It's ugly, hard to read, very very long and full of links and notes and tips and too many sentences about everything that can be shown just by a simple and nice example. 

I have the same hate feeling when I wanted to work with MongoDB. I never used MongoDB's docs, Instead I'm using other sites to read about MongoDB. Now Realm is in MongoDB's hands and docs are just awful. I wish MongoDB never bought Realm database.

**Comment by cbush** [comment]
Hi, thanks for your feedback on the docs. Sorry you're having difficulty with them. To help us improve, could you let us know what specifically you liked about the old docs? Thanks!

**Comment by lawgimenez** [comment]
I have the same feelings about the documentation. Last week I tried to setup MongoDB Atlas and I just got lost. I don't know where to start. After a day of trying to read the docs I just quit and paused all plans migrating to Atlas.
